app/code/Transoft/Blog/view/frontend/templates/singleBlogTmpl.phtml

```diff
@@ -0,0 +1,25 @@
+<?php
+use Transoft\Blog\Block\Bloglist;
+?>
+<?php /** @var Bloglist $block*/ ?>
```

Comment:
Current template should be assigned to Bloglist

Reply:
fixed